inventory/environ.py

```diff
@@ -95,6 +95,7 @@ def getDefaultVars():
     defaultVars["splunk"]["deployer_included"] = True if os.environ.get('SPLUNK_DEPLOYER_URL', False) else False
     defaultVars["splunk"]["indexer_cluster"] = True if os.environ.get('SPLUNK_CLUSTER_MASTER_URL', False) else False
     defaultVars["splunk"]["search_head_cluster"] = True if os.environ.get('SPLUNK_SEARCH_HEAD_CAPTAIN_URL', False) else False
+    defaultVars["splunk"]["search_head_cluster_url"] = os.environ.get('SPLUNK_SEARCH_HEAD_CAPTAIN_URL', None) 
```

Comment:
shouldn't this default to [sh1]:8089/path?

Reply:
Would this be clearer as `defaultVars["splunk"]["search_head_captain_url"]`?

Reply:
According to the Docs the env variable SPLUNK_SEARCH_HEAD_CAPTAIN_URL is just the hostname of the search head so we have to add the 8089/path.

Reply:
where are you adding 8089 to the path?

Reply:
The current pattern though-out is only and the hostname, this shouldn't break that pattern unless there is a clear and sensible default, rendering this optional.

To @lephino's comment, I'd say it should be
```
defaultVars["splunk"]["search_head_cluster_url"] = os.environ.get('SPLUNK_SEARCH_HEAD_CAPTAIN_URL', deaultVars["splunk"]["search_head_cluster"] + ":8089/path")
```
or something of the like...